src/Faithlife.Tracing/HttpHeaderNames.cs

```diff
@@ -0,0 +1,7 @@
+namespace Faithlife.Tracing
+{
+	public static class HttpHeaderNames
```

Comment:
I'd prefer `TracingHttpHeaderNames` because I don't like names that seem like easy conflicts with other libraries.